app/views/ReportView/components/ProbeSummary/index.tsx

```diff
@@ -161,7 +163,13 @@ const ProbeSummary = ({
     }
 
     const callSet = new ApiCallSet(apiCalls);
-    const [, reportResp] = await callSet.request(isSigned);
+    let reportResp;
+
+    if (isSigned) {
+      showConfirmDialog(callSet);
+    } else {
+      [, reportResp] = await callSet.request();
+    }
```

Comment:
ditto here, lines 174-203

missing `showConfirmDialog` in dependency array on line 204